code-challenges.test.js

```diff
@@ -40,12 +88,58 @@ var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 
 // b) Create the function that makes the test pass.
 
-
+//Declare function with arrow syntax and parameter
+//const oddOrder = (array) => {
+  //use conditional statement with `typeof()`, `.filter()`, and `%2` to find odd numbers and Mathmin() to arrange them in least to greatest...with some thought I will make two arrays to filter out numbers and then odd. Had to go back to the drawing board. More research
+  // if (typeof(value) === "number") {
+  // // let oddArray = numArray.filter(value%2 !== 0); Not working...more research.
+  //***output the array in ascending order using Mathmin...Woops. I need sort instead for a full array. Research---->`.sort(function(a, b){return a-b})` will give an ascending order for numbers. Using that anonymous function
+  // return array.sort(function(a, b){return a-b})};
+  //So this variable only removed the even numbers without the typeof conditional statement...
+//   let oddArray = array.filter( function(value){
+//     if (typeof value === "number") {return value%2 !== 0};
+//     } );
+//   //Making a variable to remove strings and `true`
+// //  let oddArray = noEven.filter(element => typeof element !== "string");
+//   return oddArray.sort(function(a, b){return a-b});
+// }
+// console.log(oddOrder(fullArr1));
+// console.log(oddOrder(fullArr2));
+
+// Just placing the function in its completed state. I test and researched so much. Got to practice various structures. SO here is the Minty Green PASS banner approved code.
+//Declare function with arrow syntax and array parameter
+const oddOrder = (array) => {
+  //After much battle I constructed a variable that used an anonymous function to take in an array an filter out only numbers and return only the odd. True and the strings only disappeared after placing a conditional statement in the anonymous function.
+  let oddArray = array.filter( function(value){
+    if (typeof value === "number") {return value%2 !== 0};
+    } );
+      //return the array in ascending order...much research on the sort function
+      return oddArray.sort(function(b, c){return b-c});
+}
```

Comment:
Nicely done! A good way to practice taking things from the internet is refactoring the code into away you are more familiar with. So if that is arrow syntax then you may want to convert the syntax and it'll help stick in your mind a bit better.